File: Documentation/5.4/Samples/python/ClientApi/Session/HowTo/Evict.py
from examples_base import ExampleBase, Employee


class Foo:
    # region evict_1
    def evict(self, entity: object) -> None: ...

    # endregion


class EvictExample(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_evict(self):
        with self.embedded_server.get_document_store("Evict") as store:
            with store.open_session() as session:
                # region evict_2
                employee_1 = Employee(first_name="John", last_name="Doe")
                employee_2 = Employee(first_name="Joe", last_name="Shmoe")

                session.store(employee_1)
                session.store(employee_2)

                session.advanced.evict(employee_1)

                session.save_changes()  # only 'Joe Shmoe' will be saved
                # endregion

            with store.open_session() as session:
                # region evict_3
                employee = session.load("employees/1-A")  # loading from server
                employee = session.load("employees/1-A")  # no server call
                session.advanced.evict(employee)
                employee = session.load("employees/1-A")  # loading form server
                # endregion

File: Documentation/5.4/Samples/php/ClientApi/session/HowTo/Evict.php
<?php

use RavenDB\Documents\DocumentStore;

interface FooInterface
{
    # region evict_1
    public function evict(object $entity): void;
    # endregion
}

class Evict
{
    public function example(): void
    {
        $store = new DocumentStore();
        try {
            $session = $store->openSession();
            try {
                # region evict_2
                $employee1 = new Employee();
                $employee1->setFirstName("John");
                $employee1->setLastName("Doe");

                $employee2 = new Employee();
                $employee2->setFirstName("Joe");
                $employee2->setLastName("Shmoe");

                $session->store($employee1);
                $session->store($employee2);

                $session->advanced()->evict($employee1);

                $session->saveChanges(); // only 'Joe Shmoe' will be saved
                # endregion
            } finally {
                $session->close();
            }

                $session = $store->openSession();
                try {
                # region evict_3
                $employee = $session->load(Employee::class, "employees/1-A");//loading from serer
                $employee = $session->load(Employee::class, "employees/1-A"); // no server call
                $session->advanced()->evict($employee);
                $employee = $session->load(Employee::class, "employees/1-A"); // loading form server
                # endregion
            } finally {
                $session->close();
            }
        } finally {
            $store->close();
        }
    }
}


class Employee
{
    private ?string $firstName = null;
    private ?string $lastName = null;

    public function getFirstName(): ?string
    {
        return $this->firstName;
    }

    public function setFirstName(?string $firstName): void
    {
        $this->firstName = $firstName;
    }

    public function getLastName(): ?string
    {
        return $this->lastName;
    }

    public function setLastName(?string $lastName): void
    {
        $this->lastName = $lastName;
    }
}
